section2/exercises/decision-making.js

```diff
@@ -38,16 +38,29 @@ if (bearChoice === 1) {
 Questions
 
 1. In English, using technical vocabulary, describe what is happening between lines 14 and 18.
+This is an if..else statement. It is reassigning bearClothing based on the value of doorChoice.
+If this condition is true, it will execute the first set of code: bearClothing = "hat"
+If this condition is false and doorChoice does not strictly equal 1, it will execute the second set of code: bearClothing = "scarf"
 
 2. What variable has a new value assigned to it after the first if statement executes?
+bearClothing
 
 3. If you changed the variable doorChoice to equal 3, what would the bearClothing value be?
+scarf
 
 4. In English, using technical vocabulary, describe what is happening between lines lines 27 and 35.
+This is an if statement that evaluates the value of bearChoice and logs specific text to the console depending on which condition is met.
+Ihe first condition being evaluated on line 27 is if bearChoice strictly equals 1, if this is true print out the line "You offer the bear your " + bearClothing + " and the bear shows you a secret passage out!"
```

Comment:
Great work on these, the specificity you went into and use of technical vocabulary is 💯